Source/Documentation/Manual/timetable.rst

```diff
@@ -2205,6 +2228,111 @@ One frame per second relates to a rotation speed of 0.1 degrees per second.
 This parameter is optional. If not defined, a default value of 30 frames per 
 second is used, which gives a default rotation speed of 3 degrees per second.
 
+.. _weather-general:
+
+Changing Weather
+================
+
+The current cloud cover, precipitation, and visibility can be varied over the 
+course of the timetable day with weather files. Weather files reside within a 
+special ``WeatherFiles`` subdirectory of the route's folder and they have the 
+file extension ``*.weather-or``. They are selected by the player from the 
+timetable mode menu.
+
+A weather file is a JSON file that consists of a single array, named "Changes," 
+each item of which represents a weather event that activates at a specific time. 
+Each event is a JSON object whose "Type" property identifies the kind of weather 
+event. Concretely, a weather file follows the format:
+
+.. code-block:: json
+
+    {
+        "Changes": [
+            {
+                "Type": "<type>",
+                "<property>": <value>,
+                ...
+            },
+            ...
+        ]
+    }
+
+There are three types of events: ``Clear``, ``Precipitation``, and ``Fog``, 
+each with their own individual sets of properties.
+
+"Clear" event
+-------------
+
+A Clear event removes any precipitation or fog while also setting the prevailing 
+overcast conditions. Clear events contain the following JSON properties:
+
+==================== ====== ===========
+Property             Type   Description
+==================== ====== ===========
+Time                 string The 24-hour time this event activates.
+Overcast             number The overcast intensity as a percentage from 0 to 100.
+OvercastVariation    number The variation in overcast intensity as a percentage from 0 to 100.
+OvercastRateOfChange number The rate of change of overcast intensity as a scaling factor from 0 to 1.
+OvercastVisibility   number The resulting visibility, in meters. The value must be in the range from 10000 to 60000. (For lower values, use a Fog event.)
+==================== ====== ===========
+
+"Precipitation" event
+---------------------
+
+A Precipitation event represents a rain spell followed by a clear spell, with 
```

Comment:
About "spell", do you mean something like a phase?

Reply:
In English, "spell" also means a period of weather. It's also the term that @Roeterdink used in the weather code.

Reply:
Alright, thank you.